application/utils/spreadsheet.py

```
@@ -91,11 +91,14 @@ def read_spreadsheet(
         logger.error('Error opening spreadsheet "%s" : "%s"' % (alias, url))
         logger.error(ae)
         exit(1)
-    except gspread.exceptions.GSpreadException as gse:
-        logger.error(
-            "If this exception says you have a duplicate cell name, the duplicate is",
-            findDups(wsh.row_values(1)),
-        )
+    except (gspread.exceptions.GSpreadException, ValueError) as gse:
+        if wsh is not None:
+            logger.error(
```

Ruff (0.15.20)

[error] Too many arguments for `logging` format string

(PLE1205)